core/iwasm/aot/aot_loader.c

```diff
@@ -1377,17 +1431,46 @@ load_import_funcs(const uint8 **p_buf, const uint8 *buf_end, AOTModule *module,
             set_error_buf(error_buf, error_buf_size, "unknown type");
             return false;
         }
+
```

Comment:
Could u give me a quick introduction about linking functions in AOT_CUSTOM_SECTION_NATIVE_SYMBOL with functions in other modules?

Reply:
This deliver does not handle native funciton refered by AOT_CUSTOM_SECTION_NATIVE_SYMBOL.
and could you let me know what is the scenario for AOT_CUSTOM_SECTION_NATIVE_SYMBOL with functions in other modules?

Reply:
"import" function either from "env"(or "wasi_unstable") module, or from other non-native modules(which are so called multi-modules) , we just check the "import" is from these non-native modules.
and the logic refer to https://github.com/bytecodealliance/wasm-micro-runtime/blob/main/core/iwasm/interpreter/wasm_loader.c#L1106

Reply:
But do we still need to search in other modules in `load_native_symbol_section` when processing `AOT_CUSTOM_SECTION_NATIVE_SYMBOL` section?

unless we are thinking about sharing native function symbols in wasm modules. But I guess we can achieve that by runtime implementation.

Reply:
seems currently AOT_CUSTOM_SECTION_NATIVE_SYMBOL only defines the reference in custom section for native symbol, no aditional functions or other components in the section, for example, the reference to index of native function for XIP indirectly calling. I think we would not share things in custom section.